docs/resolution-errors.md

```diff
@@ -0,0 +1,99 @@
+# did:webvh Resolution Error Codes
+
+This page defines the set of structured error codes and messages used in `did:webvh` DID resolution metadata, specifically in the `problemDetails` field as defined by [RFC 9457](https://datatracker.ietf.org/doc/html/rfc9457).
+
+## Using RFC 9457 in did:webvh
+
+When DID resolution fails, `did:webvh` resolvers **MUST** return an `error` field in the `didResolutionMetadata`, as defined in the [DID Resolution specification](https://www.w3.org/TR/did-core/#did-resolution). To provide additional structured information, resolvers **SHOULD** include a `problemDetails` object conforming to RFC 9457. For example, the DID Resolution Metadata would look like this:
+
+```json
+"didResolutionMetadata": {
+  "error": "invalidDid",
+  "problemDetails": {
+    "type": "https://w3id.org/security#INVALID_CONTROLLED_IDENTIFIER_DOCUMENT_ID",
+    "title": "The resolved DID is invalid.",
+    "detail": "Parse error of the resolved DID at character 3, expected ':'."
+  }
+}
+```
+
+The `problemDetails` object includes the following fields:
+
+- `type`: A URI pointing to this page, with a fragment identifier representing the error code from the [list below](#resolution-error-codes). For example:  
+  `https://didwebvh.info/main/resolution-errors/#did-log-id-mismatch`
+- `title`: A human-readable title for the error, matching the one listed in the section for the corresponding error code below.
+- `detail`: A more specific message describing the error encountered, suitable for developers or technical users.
+
+The list of `title` and `detail` values collected by `did:webvh` resolver implementers is [provided below](#resolution-error-codes).
+
+Resolver implementers **SHOULD** use the `type` and `title` values below and **MAY** use the `detail` or one appropriate for the implementation.
```

Comment:
I think the `type` should be a MUST, as this is what a test suite would likely evaluate against when asserting a negative test case response

Reply:
It doesn’t matter because this is the information site, not the spec, and the spec does not reference the extended error codes, because there is a not an approved standard to build on to do that.  The negative test case assertions such see if there is extended information about the failure available, but should accept the simpler, more or less “pass/fail” codes in the specification.